.github/ISSUE_TEMPLATE/--content-proposal-.md

```diff
@@ -6,24 +6,110 @@ labels: ''
 assignees: ''
 ---
 
+<!--
+Thank you for wanting to contribute content to amp.dev.
+
+Please replace the ✍️ HTML comments with your clear and concise responses.
+-->
+
 ## 📝 Content Proposal
 
-### Synopsis  
-A clear and concise description of the content you will be contributing.
-
-### Type of contribution  
-[ ] Guide  
-[ ] Tutorial  
-[ ] Example  
-[ ] Translation  
-[ ] Other (please specify)
-
-### Process  
-Read the [contributing guide](https://amp.dev/documentation/guides-and-tutorials/contribute/contribute-documentation/?format=websites), then create a copy of [this template](https://docs.google.com/document/d/18i_lbQ-h-auYW2SwbWRuuu2_xhtU5XrjXlcCtj0sNeY/edit?usp=sharing) and fill in the proposal and outline sections. When you’ve finished, add the link to this issue and check the box for step one. A member of the AMP team will assign the ticket to themselves and work as your reviewer through the remaining steps. Thanks for contributing to AMP! 
```

Comment:
The template from that link is now in this issue template. Not sure if that's preferable. Forgot to re-link the contributing guide.